flight/Modules/UAVOFrSKYSensorHubBridge/UAVOFrSKYSensorHubBridge.c

```diff
@@ -256,6 +254,9 @@ static void uavoFrSKYSensorHubBridgeTask(void *parameters)
 	GPSPositionData gpsPosData;
 	BaroAltitudeData baroAltitude;
 	FlightStatusData flightStatus;
+
+	uint8_t *serial_buf = frsky_state->serial_buf;
```

Comment:
Ah, I see. If you don't think it's too much of a distraction, I think the code will be cleaner, esp. since all those lines are already being changed in this PR so it won't distract from review.

P.S. Strange to me that the compiler won't optimize out this kind of thing. Place-holder variables seem like some of the easiest low-hanging fruit.